pkg/asset/machines/master.go

```diff
@@ -114,9 +135,108 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 	default:
 		return fmt.Errorf("invalid Platform")
 	}
+
+	machines := []interface{}{}
+	if m.Machines == nil {
+		for _, machine := range m.MachinesDeprecated {
+			machines = append(machines, machine)
+		}
+	} else {
+		for _, machine := range m.Machines {
+			machines = append(machines, machine)
+		}
+	}
+
+	for i, machine := range machines {
+		data, err := yaml.Marshal(machine)
+		if err != nil {
+			return errors.Wrapf(err, "marshal master %d", i)
+		}
+
+		m.FileList = append(m.FileList, &asset.File{
+			Filename: filepath.Join(directory, fmt.Sprintf(MasterMachineFileName, strconv.Itoa(i))),
```

Comment:
Can you change the `%s` to `%v` in `MasterMachineFileName` and get rid of the `strconv.Itoa` call?

Reply:
> Can you change the `%s` to `%v`...

:man_facepalming: Done with d5c01c0 -> 1c0796a.